sycl/doc/CompilerAndRuntimeDesign.md

```diff
@@ -12,21 +12,38 @@ DPC++ application compilation flow:
 
 ![High level component diagram for DPC++ Compiler](images/Compiler-HLD.svg)
 
+<div align="center"> Diagram 1. Application build flow. </div>
```

Comment:
@kbobrovs can you please check that HTML inside Markdown does not break Sphinx?

```bash
sudo apt-get install -y doxygen graphviz ssh ninja-build
sudo pip3 install 'sphinx==3.0.0' 'recommonmark==0.6.0' 'sphinx_markdown_tables==0.0.12'
python $DPCPP_HOME/llvm/buildbot/configure.py -w $DPCPP_HOME \
        -s $DPCPP_HOME/llvm -o $DPCPP_HOME/build -t Release --docs
cmake --build . --target docs-sycl-html
cd build/tools/sycl/doc/html
```

(You can find full script here: https://github.com/intel/llvm/blob/sycl/.github/workflows/gh_pages.yml)

Reply:
@alexbatashev, works for me:
```
$ cmake --build . --target docs-sycl-html
[1/1] Generating html Sphinx documentation for sycl into "/ws/sycl/build-doc/tools/sycl/doc/html"
/usr/local/lib/python3.6/dist-packages/recommonmark/parser.py:75: UserWarning: Container node skipped: type=document
  warn("Container node skipped: type={0}".format(mdnode.t))
$ echo $?
0
```